am-lib/src/main/java/uk/gov/hmcts/reform/amlib/utils/PairEntry.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.amlib.utils;
+
+import java.util.Map;
+
+@SuppressWarnings("PMD")
+//complains about setValue having a return. It is implemented this way in Map.Entry
+public class PairEntry<K, V> implements Map.Entry<K, V> {
```

Comment:
Maybe just `Pair`?

Reply:
PMD complaints. I guess this is caught under supress warnings now. Shall refactor and add into comments